src/o365/teams/commands/report/teams-report-useractivityuserdetail.ts

```diff
@@ -49,8 +55,39 @@ class TeamsReportUserActivityUserDetailCommand extends GraphCommand {
     request
       .get(requestOptions)
       .then((res: any): void => {
-        cmd.log(res);
-
+        if (args.options.outputFile) {
```

Comment:
The `--outputFile` check should come after checking the `--output` option. We should be able to print json on the screen in case of `json` being specified in the `--output` or save that json to a file in case `--outputFile` is specified. Something similar is done by Waldek here: https://github.com/pnp/office365-cli/pull/1037/files

Reply:
I have fixed this in the latest commit. Just an FYI here user can export the report data to the specified path in csv,txt and json formats